opam.export

```diff
@@ -178,8 +191,10 @@ installed: [
   "ocaml-syntax-shims.1.0.0"
   "ocaml-version.3.4.0"
   "ocamlbuild.0.14.0"
+  "ocamlc-loc.3.5.0"
   "ocamlfind.1.9.3"
   "ocamlformat.0.20.1"
+  "ocamlformat-rpc-lib.0.26.1"
```

Comment:
Not needed technically but ocaml lsp always complains that it can't find it so OK I suppose.